langchain/src/chains/chat_vector_db_chain.ts

```diff
@@ -11,12 +11,12 @@ import { CallbackManagerForChainRun } from "../callbacks/manager.js";
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export type LoadValues = Record<string, any>;
 
-const question_generator_template = `Given the following conversation and a follow up question, rephrase the follow up question to be a standalone question.
```

Comment:
This one is deprecated, no need to modify it

Reply:
Yes, that makes sense! Let me remove this change from this file

Reply:
Done ;)